relay/kusama/src/lib.rs

```diff
@@ -1919,7 +1922,7 @@ impl Runtime {
 		};
 
 		// We assume un-delayed 6h eras.
-		let era_duration = 6 * HOURS;
+		let era_duration = 6 * (HOURS as Moment) * MILLISECS_PER_BLOCK;
```

Comment:
unfortunately there was a small mistake here as well, but since it is read-only and experimental, it is fine.